go/api/generic_commands_test.go

```diff
@@ -16,3 +18,26 @@ func ExampleGlideClient_CustomCommand() {
 
 	// Output: PONG
 }
+
+func ExampleGlideClient_Scan() {
+	var client *GlideClient = getExampleGlideClient() // example helper function
+	resCursor, resCollection, err := client.Scan(0)
+	if err != nil {
+		fmt.Println("Glide example failed with an error: ", err)
+	}
+	fmt.Println("Scan Cursor: ", resCursor, "Collection: ", resCollection)
+
+	// Output: Cursor: [0],  Collection: [key456 key123 key789 keylist1]
```

Comment:
This will always fail. Cleanup DB and create few keys to ensure that `SCAN` returns constant response